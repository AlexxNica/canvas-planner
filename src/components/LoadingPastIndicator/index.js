import React, { Component } from 'react';
import Spinner from 'instructure-ui/lib/components/Spinner';
import formatMessage from 'format-message';

export default class LoadingPastIndicator extends Component {
  render () {
    return <div>
      <Spinner size="small" title={formatMessage('Loading past items...')}/>
      {formatMessage('Loading Past Items...')}
    </div>;
  }
}
