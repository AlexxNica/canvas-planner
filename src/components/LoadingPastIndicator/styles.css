.root {
  display: flex;
  align-items: center;
  justify-content: center;
  margin-top: var(--margin);
  margin-bottom: var(--margin);
}

.message {
  font-size: var(--fontSize);
  font-family: var(--fontFamily);
  font-weight: var(--fontWeight);
  margin-left: var(--messageMargin);
  color: var(--color);
}
