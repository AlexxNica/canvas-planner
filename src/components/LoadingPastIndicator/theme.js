export default function generator ({ colors, typography, spacing }) {
  return {
    fontSize: typography.fontSizeSmall,
    fontFamily: typography.fontFamily,
    fontWeight: typography.fontWeightNormal,
    color: colors.slate,
    messageMargin: spacing.xSmall,
    margin: spacing.xSmall
  };
}
