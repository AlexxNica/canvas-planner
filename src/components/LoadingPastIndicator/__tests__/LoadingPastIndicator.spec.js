import React from 'react';
import LoadingPastIndicator from '../index';
import {shallow} from 'enzyme';

it('renders stuff', () => {
  const wrapper = shallow(<LoadingPastIndicator />);
  expect(wrapper).toMatchSnapshot();
});
