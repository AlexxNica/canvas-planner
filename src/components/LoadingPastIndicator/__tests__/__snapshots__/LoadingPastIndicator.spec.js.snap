// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders stuff 1`] = `
<div>
  <Spinner
    size="small"
    title="Loading past items..."
    variant="default"
  />
  Loading Past Items...
</div>
`;
